modulefiles/modulefile.ww3.orion.lua

```diff
@@ -23,3 +23,4 @@ load(pathJoin("hdf5", "1.10.6"))
 load(pathJoin("netcdf", "4.7.4"))
 
 load(pathJoin("w3nco", "2.4.1"))
```

Comment:
I think you can also remove w3nco.  Do you have paths to the output of your coupled model run by any chance I could look at?

Reply:
I think so too.
`w3nco` is no longer required in any library/executable, and if it is, we need to snuff it out.
I know it is being automatically loaded when `nemsio/2.5.2` is loaded.  It was removed from `nemsio/2.5.3+`

Reply:
Sorry, my wires were crossed. I was thinking of the thing I added, but this is about removing the thing it basically replaced. Doing a test build just to be sure, but should work.

Reply:
Done